File: MSimpleBoard/src/main/java/com/mooop/board/repo/UserRepository.java
package com.mooop.board.repo;

import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.Query;

import com.mooop.board.entity.MSBUser;

public interface UserRepository extends JpaRepository<MSBUser, Long>{
	
	MSBUser findByUserNick(String nick);
	
}

File: kmvc/security/src/main/kotlin/com/mooop/sec/common/repository/UserRepository.kt
package com.mooop.sec.common.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/**
 * Project : kmvc
 * Package :com.mooop.sec.common.repository
 * Author :  zinnaworks
 * Date : 09/04/2022
 * Desc :
 */
@Repository
interface UserRepository:JpaRepository<UserEntity , Long> {
    fun findByEmail(email:String):UserEntity?
}
